langchain4j-dashscope/src/main/java/dev/langchain4j/model/dashscope/QwenHelper.java

```diff
@@ -388,8 +370,8 @@ static ToolBase toToolFunction(ToolSpecification toolSpecification) {
     private static JsonObject toParameters(ToolSpecification toolSpecification) {
         if (toolSpecification.parameters() != null) {
             return JsonUtils.toJsonObject(toMap(toolSpecification.parameters()));
-        } else if (toolSpecification.toolParameters() != null) {
-            return JsonUtils.toJsonObject(toolSpecification.toolParameters());
+        } else if (toolSpecification.parameters() != null) {
+            return JsonUtils.toJsonObject(toolSpecification.parameters());
```

Comment:
This does not seem right, `toolSpecification.parameters()` is already handled above.

We should keep `toolSpecification.parameters()` support for backwards-compatibility